test/functional/wallet_migration.py

```diff
@@ -1020,6 +1021,53 @@ def test_migrate_simple_watch_only(self):
         assert_equal(wo_wallet.listdescriptors()['descriptors'][0]['desc'], descsum_create(f'pk({pubkey.hex()})'))
         wo_wallet.unloadwallet()
 
+    def test_manual_keys_import(self):
+        self.log.info("Test migrating standalone private keys")
+        wallet = self.create_legacy_wallet("import_privkeys", blank=True)
+        privkey, pubkey = generate_keypair(wif=True)
+        wallet.importprivkey(privkey=privkey, label="hi", rescan=False)
+        wallet.unloadwallet()
+
+        # Now migrate wallet in the latest node
+        res = self.master_node.migratewallet(self.old_node.wallets_path / "import_privkeys")
```

Comment:
In 080e7ec110b0e40e56b9aad608cd144d75b971f7 "test: add coverage for migrating standalone imported keys"

Use `self.migrate_and_get_rpc`